apps/molgenis-components/src/components/forms/FormGroup.vue

```diff
@@ -72,13 +74,25 @@ export default {
       <label for="my-id2">required field form group</label>
       <FormGroup id="my-id2"
         label="my label"
-        required
+        required="'true'"
```

Comment:
use ':required' here, without the colon the inner quotes are part of the string

Reply:
The old version would also still be a good example (just passing `required` is shorthand for `:required="true"`)